docs/meshcloud.payment-methods.md

```diff
@@ -0,0 +1,94 @@
+
+## The Payment Method Lifecycle: Assigning to Projects
+
+Now that the Partner Admin has created one or more payment methods for a meshCustomer, we are ready to link a payment method to one or more projects.
+
+First, check if the payment method is correctly created in the meshCustomer. You can do so by navigating to the meshCustomer Account area. In order to do that you need to have Customer Admin rights within the respective customer. On the left-hand side, there should be an item called 'Payment Methods' which you can navigate to. All payment methods that are created and assigned to your current meshCustomer are shown here.
+
+The payment methods that are assigned can be used for both existing projects and new projects.
+
+### Applying a payment method to an existing project
+
+In the Customer Account area, open 'Projects' on the left. Then click on the 'Edit project' button on the right for the project you would like to add a payment method. In the 'Edit Project' screen, you'll see the selected payment methods on the right. As explained at the top of the page, you also have the ability to specify a Substitute Payment Method. A Substitute Payment Method is useful when working with expiring cost centers or budgets. This makes sure that when the active payment method expires, it will activate the substitute payment method.
+
+![Set Payment Method in Project Edit screen](assets/payment_methods/payment_method_selection_project_edit.png)
+
+Select the Active Payment Method of choice, and optionally a Substitute Payment Method if that is desired. Save the project and the payment method(s) will be assigned to the project.
+
+### Applying a payment method to a new project
+
+Create a new project at the top of the meshPanel. Follow the project creation flow until you enter the 'Billing Information' page.
+
+![Set Payment Method in Project Create screen](assets/payment_methods/payment_method_selection_project_create.png)
+
+As you can see, on this page you have the ability to assign a payment method to the project. Keep in mind that this is mandatory. If you want to specify a substitute payment method, you can do so after creating the project and following the steps above in '**Applying a payment method to an existing project'**. If your meshCustomer has no payment methods, you will be confronted with a warning that looks like this:
+
+![Missing Payment Method warning](assets/payment_methods/payment_method_missing_in_project_create.png)
+
+After setting a payment method, you can continue the project creation flow and save the new project. You now have a new project with a payment method successfully assigned!
+
+## The Payment Method Lifecycle: Enhance with Metadata
```

Comment:
Especially here I think that `lifecycle` does not fit as a term. Usually you cannot skip steps in lifecycles. Also this enhancing with metadata you can basically do always once the meshPaymentMethod is created i think.